src/execution_plan/ops/op_procedure_call.c

```diff
@@ -139,10 +166,17 @@ static void ProcCallFree(OpBase *ctx) {
 	}
 
 	if(op->args) {
+		for(uint i = 0; i < op->arg_count; i++) SIValue_Free(op->args + i);
```

Comment:
As on 124, this can free uninitialized values - `array_len` would be safe.

Reply:
👍